src/Diff/index.js

```diff
@@ -25,7 +25,11 @@ const setUserSelectStyle = (element, selectable) => {
 const Diff = props => {
     const root = useRef(null);
     const enableColumnSelection = useCallback(
-        ({target}) => {
+        ({ target, button }) => {
```

Comment:
Thanks for this great PR, however to align with our coding style, you have to remove spaces inside  and modify it to`({target, button}) => {`

Reply:
@otakustay done